### ISSUE TITLE
CUDA Download link needs update

### ISSUE THREAD

**Issue by MovGP0** [tensorflow/tensorflow#31143]
**Describe the current behavior**

When CUDA is not installed, I get the following error

```
...\tensorflow\python\platform\self_check.py in preload_check()

ImportError: Could not find 'cudart64_100.dll'. TensorFlow requires that this DLL be installed in a directory that is named in your %PATH% environment variable. Download and install CUDA 10.0 from this URL: https://developer.nvidia.com/cuda-90-download-archive
```

The link is for CUDA 9.0 rather than 10.0. 

**Describe the expected behavior**

Use one of these links (whichever is appropriate): 

```
https://developer.nvidia.com/cuda-10.0-download-archive
https://developer.nvidia.com/cuda-10.1-download-archive
```

**Comment by MovGP0** [comment]
btw: installed CUDA from `https://developer.nvidia.com/cuda-10.0-download-archive`, installed it, and restarted the computer. The path to the `%programfiles%\NVIDIA GPU Computing Toolkit\CUDA\v10.0\bin` directory was added in the system-wide path variable. 

Whent typing 

```cmd
where cudart64_100.dll
```

I get 

```cmd
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.0\bin\cudart64_100.dll
```

However, I still get the above error in my Jupyter Notebook.

**Comment by anush-o** [comment]
@MovGP0 ,
Just to clarify did you add sub folders: bin and lib to the path?Thanks!

**Comment by ymodak** [comment]
@MovGP0 You are right. Thanks for reporting this.  What version of TF you are trying to install?

**Comment by MovGP0** [comment]
@ymodak v1.14 GPU 
```
pip install tensorflow-gpu==1.14
```

**Comment by MovGP0** [comment]
@anush-o the bin and lib folders where added to the path variable during the Installation of CUDA.

**Comment by ymodak** [comment]
@yifeif Can you please take a look? Thanks!

**Comment by mohapatras** [comment]
> btw: installed CUDA from `https://developer.nvidia.com/cuda-10.0-download-archive`, installed it, and restarted the computer. The path to the `%programfiles%\NVIDIA GPU Computing Toolkit\CUDA\v10.0\bin` directory was added in the system-wide path variable.
> 
> Whent typing
> 
> ```batchfile
> where cudart64_100.dll
> ```
> 
> I get
> 
> ```batchfile
> C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.0\bin\cudart64_100.dll
> ```
> 
> However, I still get the above error in my Jupyter Notebook.


@MovGP0 This occurred to me. My setting was CUDA 10.0, cudNN 7.4.2, tensorflow 1.14. I had the same issue mutliple times. I have CUDA 9.0 installed too. In environment variables, I changed the ordering in "Variable name" "path". Moved up v10.0 path, libnvvp and moved the cuda 9.0 below. Restarted the system.

It worked. I have attached a screenshot in the link.
[Image](https://drive.google.com/open?id=1QdBQ8paxecS-wNrAyd6uynxNv1ZDkKWe)

**Comment by ymodak** [comment]
The error message is fixed with TF 1.15 version. Thanks!
